### PR TITLE
hypervisor: switch to Q35

### LINKED ISSUE #706: switch to q35
we need to switch to Q35, since this machine type has more features and better performance

### PR DESCRIPTION
**Pull request by devimc**
Fixes #706

### CODE REVIEW

**Comment by devimc** [comment]
depends on https://github.com/clearcontainers/runtime/pull/705

**Comment by devimc** [comment]
depends on https://github.com/clearcontainers/packaging/pull/34

File: config.go [hunk 1]
```diff
@@ -135,6 +137,23 @@ func (h hypervisor) image() (string, error) {
 	return resolvePath(p)
 }
 
+func (h hypervisor) firmware() (string, error) {
```
Comment (jodh-intel) [inline config.go]: This needs a test.

File: config.go [hunk 2]
```diff
@@ -135,6 +137,23 @@ func (h hypervisor) image() (string, error) {
...
+		if defaultFirmwarePath == "" {
+			return "", nil
+		}
+		p = defaultFirmwarePath
+	}
+
+	return resolvePath(p)
+}
+
+func (h hypervisor) machineAccelerators() string {
```
Comment (jodh-intel) [inline config.go]: Another one that needs a very basic test.

**Comment by devimc** [comment]
depends on https://github.com/ciao-project/ciao/pull/1536

**Comment by devimc** [comment]
depends on https://github.com/containers/virtcontainers/pull/456

**Pull request closed without merging**